app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,7 @@
     <main class="px4">
       <div class="flash-message">
         <% flash.each do |name, msg| %>
-          <%= content_tag :div, msg, class: name %>
+          <%= content_tag :div, msg.html_safe, class: name %>
```

Comment:
Way to fix this flash message Melanie. I was calling html_safe, but in the controller,  good to know it works if its in the application file.